nservicebus/operations/system-resources.md

```diff
@@ -0,0 +1,102 @@
+- What Platform components are used (i.e., [error instance](/servicecontrol/servicecontrol-instances/), [audit instance](/servicecontrol/audit-instances/), [monitoring instance](/servicecontrol/monitoring-instances/), etc)
+- Whether the [MassTransit Connector](/servicecontrol/masstransit/) is used
+- Whether any legacy [transport adapter](/servicecontrol/transport-adapter) is used
+- Whether the [messaging bridge](/nservicebus/bridge/) is used, how many bridges have been implemented, and for each one of them:
+  - What are the endpoints they are responsible for bridging
+  - What are the transports they are supporting
+- Whether any [gateway](/nservicebus/gateway/) is used and their configured  persistence type
+- What [community components](/nservicebus/community/) are used
+
+For each endpoint that needs to be deployed, the following information must be collected to successfully deploy systems without using installers:
+
+- Name of the endpoint
+- If the endpoint is configured to be [uniquely addressable](/nservicebus/messaging/routing#make-instance-uniquely-addressable)
+- Queues required by the endpoint (i.e., input queue, error queue, audit queue, queues for metrics, queues for heartbeats, etc.)
+- Subscriptions created by the endpoint (i.e., subscription for every event type, manually created subscriptions, etc.)
+- Whether outbox is used
+- Whether auditing is enabled
+- Whether saga auditing is enabled
+- Whether satellites are used
+- Message types processed by this endpoint (including inheritance hierarchy, dynamically laoded types, and assembly scanning)
+
+Transport specific:
+
+- Topology (i.e., bundle topics, hierarchy bundle, filtering for subscriptions, etc.)
+- Attributes of the infrastructure (i.e., quotas, delivery count, TTL, etc.)
+- Attributes of the subscription (i.e., topics, table prefixes, table names, etc.)
+- Queues required by the transport (i.e., timeout queues, delay queues, dead letter queues, etc.)
+- Technology-specific queues (i.e., queues for delays that differ between versions, queues for quorum, etc.)
+
+Persistence specific:
+
+- Outbox details (i.e., prefixes and table names, etc.)
+- Saga details (i.e., prefixes and table names, etc.)
+- (Legacy) Timeout details (i.e., prefix and table names for timeout manager, etc.)
+- (Legacy) Subscription cache (i.e., prefix and table names, etc.)
+
+Platform specific:
+
+- Main instance:
+ - Queues (i.e., input queue, error queue, error forwarding, saga audit queue, audit forwarding queue, queue used for Service Pulse, acknowledgement queue, staging queue for Edit & Retry, throughput data, internal errors)
+ - Databases (i.e., RavenDB instances)
+- Audit instance
+ - Queues (i.e., input queue)
+- Metrics instance
+ - Queues (i.e., input queue)
+
+MassTransit connector specific:
+
+- Queues used by MassTransit (i.e., the queues on "the other end")
+- Error queues
+- Connector-specific queues (i.e., return queue, poison queue, etc.)
```

Comment:
- The [return queue](https://docs.particular.net/servicecontrol/masstransit/#what-queues-are-created) used by the Mass Transit connector
- The [platform specific](#platform-specific-details) infrastructure